ECHO-COVID19.ipynb

```diff
@@ -1121,6 +965,10 @@
     "    \n",
     "    # Merge ECHO and NPDES data\n",
     "    latest = latest.join(echo_data)\n",
+    "    \n",
+    "    # Drop duplicates to just map a facility once\n",
+    "    latest = latest.loc[~latest.index.duplicated(keep='first')]\n",
+    "    \n",
     "    print(latest)\n",
     "    \n",
```

Comment:
In the "Which facilities have officially indicated that they could not monitor and report water quality due to COVID?" section, a number out of total facilities should be printed (it prints a table and number of rows/columns)

Reply:
done